src/Plugin/views/field/OgMembershipBulkForm.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Drupal\og\Plugin\views\field;
+
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\og\OgMembershipInterface;
+use Drupal\system\Plugin\views\field\BulkForm;
+
+/**
+ * Defines a user operations bulk form element.
+ *
+ * @ViewsField("og_membership_bulk_form")
+ */
+class OgMembershipBulkForm extends BulkForm {
+
+  /**
+   * {@inheritdoc}
+   *
+   * Provide a more useful title to improve the accessibility.
+   */
+  public function viewsForm(&$form, FormStateInterface $form_state) {
+    // Remove all actions related to (non-)member role.
+    foreach ($this->actions as $key => $action) {
+      if (strstr($key, '-member') !== FALSE) {
```

Comment:
non-member, member and administrator are default roles. The first one is implied as it applies any user that is not a member of the group. Group members can be group administrators. And a group administrator is always a group member.

The role related actions are useful. You'd want to want to make someone and administrator or give them custom group roles. They should not apply to member/non-member and that is.

Will provide a diff of the membership actions shortly.